AEPConcierge/Sources/Models/ConciergeSourceReference.swift

```diff
@@ -0,0 +1,49 @@
+/*
+ Copyright 2025 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import Foundation
+
+/// A model describing a single source reference shown beneath an agent response.
+public struct ConciergeSourceReference: Identifiable, Codable, Hashable {
```

Comment:
citations currently comes as an array of strings, and it's represented in `ConciergeJsonContentObject` (future) and `TempSource` (temp...)

i'm fine with however we want to approach this, but let's make sure we don't have duplicate models :D